.github/ISSUE_TEMPLATE/improve-existing-docs.yaml

```
@@ -36,7 +36,7 @@ body:
       label: What did you expect to happen?
       description: Tell us what you think should have happened.
       placeholder: |
-          "This feature should have been documented with {X}, {Y}, and {Z} details."
+          "This feature should have been documented with details such as {X}, {Y}, and {Z}"
```

```suggestion
          "This feature should have been documented with the following details: {X}, {Y}, and {Z}"
```

How do you feel about this revision?

---

Thanks for contributing. Let me know what you think about my suggestion.

---

It's nice! I will commit it.